HW3/HW3_Khyzhniak.js

```diff
@@ -0,0 +1,133 @@
+function task1() {
+  console.log('%c --- TASK #1 --- ', 'color: green; font-weight: bold; font-size: 1.5em;');
+
+  function sum(a = 0) {
+    let sum = a;
+    
+    return a === 0 ? a :
+    function add(b) {
+      return b === undefined ? sum : (
+        sum += b,
+        add
+      )
+    }
```

Comment:
- " a" / "b" not the best names, try to give more informative names. in this case you can use "argument"
- just as a spelling option "return a ?? add"

Reply:
Agree